pkg/kdump/kdump.js

```diff
@@ -63,12 +63,18 @@ const initStore = function(rootElement) {
             stateChanging: dataStore.stateChanging,
             reservedMemory: dataStore.kdumpMemory,
             kdumpStatus: dataStore.kdumpStatus,
+            kdumpCmdlineEnabled: dataStore.crashkernel,
             onSaveSettings: dataStore.saveSettings,
             onCrashKernel: dataStore.kdumpClient.crashKernel,
         }), rootElement);
     };
     dataStore.render = render;
 
+    cockpit.file("/proc/cmdline").read()
+            .then(content => {
+                dataStore.crashkernel = content.indexOf('crashkernel=') !== -1;
+            });
```

Comment:
Is it guaranteed to be enabled in /proc/cmdline always? I.e. there's no way to set that in the kernel config?

Please handle failures. read() returns `null` if the file does not exist (yes, strange API), so calling `null.indexOf()` will crash. Permission errors will cause a rejection as usual. I know it's unlikely, but in principle we can connect to LXC or nspawn containers or have an environment where an unprivileged user can't read the file.